app/teams/create/InviteForm.tsx

```
@@ -42,198 +47,291 @@ export function InviteForm({ onBack, teamName }: InviteFormProps) {
-                    key={user.id}
-                    className="flex items-center gap-3 p-3 bg-gray-50 rounded-lg group"
+            {searchStats.selectedCount > 0 && (
+              <div className="space-y-4">
+                <div className="flex items-center justify-between">
+                  <h3 className="text-sm font-medium text-gray-700">
+                    Selected Teammates ({searchStats.selectedCount})
+                  </h3>
+                  <button
+                    onClick={handleClearAllUsers}
+                    className="text-xs text-red-600 hover:text-red-800 font-medium transition-colors"
                   >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)